Library/Formula/cmocka.rb

```diff
@@ -1,17 +1,15 @@
-require 'formula'
-
 class Cmocka < Formula
-  homepage 'http://cmocka.cryptomilk.org/'
-  url 'https://open.cryptomilk.org/attachments/download/42/cmocka-0.4.1.tar.xz'
-  sha1 'ef8b64878edc5f0e442dc6aaf3dc83ef912c51c2'
+  homepage "https://cmocka.org/"
+  url "https://cmocka.org/files/1.0/cmocka-1.0.0.tar.xz"
+  sha1 "c1b6bdab164b2d54c26edacef9b9308e4de154f9"
 
-  depends_on 'cmake' => :build
+  depends_on "cmake" => :build
 
   def install
-    mkdir 'build' do
+    mkdir "build" do
       system "cmake", "..", "-DUNIT_TESTING=On", *std_cmake_args
       system "make"
-      system "make install"
+      system "make", "install"
     end
   end
```

Comment:
Any chance a test can be added to do something more substantial than e.g. `--version` or `--help`? See `cmake.rb` for an example of an application formula with a good test and `tinyxml2.rb` for an example of a library formula with a good test. Thanks!